packages/calypso-products/src/experiments.ts

```diff
@@ -22,3 +22,13 @@ export const setPlansListExperiment = ( experimentName: string, variation: strin
 export const getPlansListExperiment = ( experimentName: string ): string | undefined => {
 	return getExperiment( PLANS_LIST_NAMESPACE, experimentName );
 };
+
+export const setTrailMapExperiment = ( variation: string ): void => {
```

Comment:
I recognize that there is an established convention here with Claudiu's previous work. Given the context, this implementation seems reasonable to me. It's also worth noting for posterity that this experiment code is temporary because of the nature of experiments.

One thing I'll mention, however, is that _we do_ also dependency inject params to determine which features to show or not to show ( as opposed to using globals like `isTrailMapVariant` ) elsewhere in this package

See examples like https://github.com/Automattic/wp-calypso/blob/trunk/packages/calypso-products/src/plans-list.tsx#L1014

Reply:
TIL! Thanks for the info Dehan 🙂

It sounds like there's a technical limitation because Webpack won't tree shake the unused object members of, for example, the `PLANS_LIST` return value of `getPlan`. Similarly, we'd see the same issue with the `FEATURES_LIST` we're updating in this PR.

I think that's a fair concern and am fine with the implementation 👍